### PR TITLE
sky - the beginning

### PR DESCRIPTION
**Pull request by ngwese**
this is (very) light on documentation but long in ambition. my intention is to:
- work up some combination of reference documentation
- start a [studies](https://github.com/ngwese/sky-studies) type series
- stabilize the norns, arc, then grid abstractions (input end is okay, output/visualization bits are still rough)

### CODE REVIEW

File: lua/extn/sky/prelude.lua [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+local function import(target, module)
+  for k, v in pairs(module) do
+    target[k] = v
+  end
+end
+
+sky = {
+  __loaded = {},
+  __search = {norns.state.path, _path.code},
```
Comment (tehn) [inline lua/extn/sky/prelude.lua:13]: remember we recently implemented the `paths` table for readability :)

Comment (ngwese) [reply lua/extn/sky/prelude.lua:13]: good point. i'll switch that.

Comment (ngwese) [reply lua/extn/sky/prelude.lua]: okay, everything uses `paths` now. thanks for catching that.

**Comment by ngwese** [comment]
ugh @tehn - GH wants approval still since i changed the code...

**Comment by ngwese** [comment]
never mind it looks like a stale www page, now merged.

**Comment by tehn** [comment]
looking good!

**Pull request merged**